distribution/docs/src/main/resources/content/_metadata-attributes/NITF-Attribute-Mappings-table-contents.adoc

```diff
@@ -0,0 +1,38 @@
+:title: NITF Attribute Mappings
+:type: subappendix
+:parent: Metadata Attributes
+:status: published
+:summary: NITF Attribute Mappings.
+
+.[[NITF_ACFTB_Attribute_Mappings]]NITF Attribute Mappings
```

Comment:
I plan to write a new ticket for reworking this table and correcting the mappings. I will link it here.